### ISSUE TITLE
Accessibility - card section does not accept aria-label property

### ISSUE THREAD

**Issue by kyjak** [carbon-design-system/carbon-for-ibm-dotcom#6159]
we are unable to provide a label to the card section simple element, there is a constant "placeholder" now
reporting url https://ibmdocs-dev.mybluemix.net/docs/en/cloud-paks
the tool to check errors is "IBM Equal Access Accessibility Checker", plugin for chrome
i would suggest that ALL dotcom components accept the basic aria properties so anybody can set them

`<div role="region" aria-label="placeholder" class="bx--card-group__card">`

**Comment by DragosRistici** [comment]
This is [an accessibility issue](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).

Read more here, too:
https://www.w3.org/TR/wai-aria/#aria-label

**Comment by DragosRistici** [comment]
The card group simple component does not accept an aria-label, it has a constant of "placeholder" set as aria-label.

**Comment by ariellalgilmore** [comment]
Hi @kyjak! Looks like you are using `Carbon for IBM.com v1.18.1`. The appropriate aria-labels have been fixed and included in `v1.19.0`. Let us know if you still are facing issues once you upgrade.

**Comment by RobertaJHahn** [comment]
Fixed with the upgrade.
